test/e2e/play_kube_test.go

```diff
@@ -4724,6 +4712,7 @@ cgroups="disabled"`), 0644)
 	})
 
 	It("podman kube --quiet with error", func() {
+		SkipIfNotRootless("We need to create an error trying to bind to port 80")
```

Comment:
This seems out of context.  Please move it to another commit (with explanation) if needed.

Reply:
So the issue here is, that this new test was relying on that playing this kubernetes yaml would create an error as it would bind two pods (due to replicas = 2) to the same port. With my PR, that would no longer error out and this test would now start failing.

I have added it to the same commit, so that every commit passes the CI and can thus be safely backported. I can move it to another commit though, if you prefer that.

Reply:
Ah, OK.  That makes sense, thank you!  No need to change anything.  Was a brain fart on my end.

Reply:
That hardly qualifies as a brain fart…